Trouble with using Ibex without FuseSoC

Hi,

I'm not sure if this is the correct approach but I was planning to gather all RTL files and run simulations/tests on QuestaSim so I found the following in your documentation:

"
Ibex flows use [FuseSoC](https://github.com/olofk/fusesoc) to gather needed RTL files and run builds. If you want to use Ibex without FuseSoC the following FuseSoC command will copy all the needed files into a build directory.

fusesoc --cores-root . run --target=lint --setup --build-root ./build/ibex_out lowrisc:ibex:ibex_top
"

I have used the command above and it did generate a bunch of files and what seems to be a makefile for verilator simulation so I thought I'd run it first to make sure I'm not missing any files.

I ran make binary as well as make lint-only and they produce the same error:

$ make binary
verilator --binary -f lowrisc_ibex_ibex_top_0.1.vc -Wall --unroll-count 72
%Error: src/lowrisc_prim_xilinx_clock_gating_0/rtl/prim_clock_gating.sv:23:7: Cannot find file containing module: 'BUFGCE'
   23 |       BUFGCE #(
      |       ^~~~~~
        ... Looked in:
             src/lowrisc_dv_dv_fcov_macros_0/BUFGCE
             src/lowrisc_dv_dv_fcov_macros_0/BUFGCE.v
             src/lowrisc_dv_dv_fcov_macros_0/BUFGCE.sv
             src/lowrisc_prim_util_get_scramble_params_0/rtl/BUFGCE
             src/lowrisc_prim_util_get_scramble_params_0/rtl/BUFGCE.v
             src/lowrisc_prim_util_get_scramble_params_0/rtl/BUFGCE.sv
             src/lowrisc_prim_util_memload_0/rtl/BUFGCE
             src/lowrisc_prim_util_memload_0/rtl/BUFGCE.v
             src/lowrisc_prim_util_memload_0/rtl/BUFGCE.sv
             src/lowrisc_prim_assert_0.1/rtl/BUFGCE
             src/lowrisc_prim_assert_0.1/rtl/BUFGCE.v
             src/lowrisc_prim_assert_0.1/rtl/BUFGCE.sv
             src/lowrisc_prim_secded_0.1/rtl/BUFGCE
             src/lowrisc_prim_secded_0.1/rtl/BUFGCE.v
             src/lowrisc_prim_secded_0.1/rtl/BUFGCE.sv
             BUFGCE
             BUFGCE.v
             BUFGCE.sv
%Error: Exiting due to 1 error(s)
make: *** [Makefile:19: binary] Error 1

Can you please let me know what I'm doing wrong? Thanks!

--- a/ibex_top.core
+++ b/ibex_top.core
@@ -19,6 +19,8 @@ filesets:
       - lowrisc:prim:onehot_check
       - lowrisc:prim:onehot
       - lowrisc:prim:util
+      - "fileset_partner ? (partner:prim_generic:all)"
+      - "!fileset_partner ? (lowrisc:prim_generic:all)"
     files:
       - rtl/ibex_register_file_ff.sv # generic FF-based
       - rtl/ibex_register_file_fpga.sv # FPGA

--- a/ibex_top_tracing.core
+++ b/ibex_top_tracing.core
@@ -9,8 +9,6 @@ filesets:
     depend:
       - lowrisc:ibex:ibex_top
       - lowrisc:ibex:ibex_tracer
-      - "fileset_partner ? (partner:prim_generic:all)"
-      - "!fileset_partner ? (lowrisc:prim_generic:all)"
     files:
       - rtl/ibex_top_tracing.sv
     file_type: systemVerilogSource

